src/OrchardCore.Modules/OrchardCore.Forms/placement.json

```diff
@@ -0,0 +1,101 @@
+{
+
+  "TitlePart_Edit": [
+    {
+      "place": "FormSettings:5",
+      "contentType": [
+        "Form",
+        "Label"
+      ]
+    }
+  ],
+
+  "FormInputElementPart_Fields_Edit": [
+    {
+      "place": "FormSettings:5",
+      "contentType": [
```

Comment:
This shouldn't have to be in placement, it should be written in such a way that it can be controlled and located by the driver (no issue with an example placement being in here, but it should match the driver settings)

Where your are having to restrict by content type is indicating (to me) that there is a flaw somewhere in the design

Reply:
I agree, However Editor shape is build using BuildEditor and when ContentCard is created - the Editor shape shape was already created and Parts are placed as per placement. 

So we transform Editor shape to ContentCard by manually adding/removing shapes. I can't find better way with current implementation.

We are already doing that for `Footer` below,  I extended same idea to target other zones

https://github.com/OrchardCMS/OrchardCore/blob/82ec408acab8d6d984741b61ce57df679a2f74aa/src/OrchardCore.Modules/OrchardCore.Widgets/Views/ContentCard.cshtml#L30-L32

Reply:
I used placement because, I want to target `FormInputElementPart_Fields_Edit`  part for `ContentType` defined in `Forms` only  and don't want to break default `Content_Edit` for other content types outside form modules that use the same Part